boot_script.sh

```diff
@@ -0,0 +1,5 @@
+
+for ITER in 1
+do
+    mix performance_test --mode autodemand --numberOfElements 30 --howManyTries 3 --tick 10000 --reductions 1000 --initalGeneratorFrequency $((2500*$ITER)) --chosenMetrics passing_time_avg,passing_time_std,throughput --shouldAdjustGeneratorFrequency --shouldProducePlots --shouldProvideMetricsHeader /project/results/
```

Comment:
```suggestion
    mix performance_test --mode autodemand --numberOfElements 30 --howManyTries 3 --tick 10000 --reductions 1000 --initalGeneratorFrequency $((2500*$ITER)) --chosenMetrics passing_time_avg,passing_time_std,throughput --shouldAdjustGeneratorFrequency --shouldProducePlots --shouldProvideMetricsHeader ./project/results/
```
btw. test still breaks when the path does not exist

Reply:
Indeed, `mkdir_p` should deal with it :P